### PR TITLE
Adding a DMS section to the how to docs

### PR DESCRIPTION
**Pull request by AwkwardBen**
Chnages to grammar and spelling

the the to the

Changes to grammar and spelling

### CODE REVIEW

File: how-to-docs/dms-migration.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+The following are the options and an example of answers for this tasks page:
+Option | Example answer | Description
+------ | -------------- | -----------
+Task name | Migration-task | A name for the task.
+Task Description | Task for migrating | A description for the task.
+Source endpoint | source-instance | The source endpoint for migration.
+Target endpoint | target-instance | The target endpoint for migration.
+Replication instance | replication-instance | The replication instance to be used.
+Migration type | Migrate existing data | Migration method you wnat to use.
```
Comment (oyelekci) [inline how-to-docs/dms-migration.md]: typo: 'want'

File: how-to-docs/dms-migration.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
+# **DMS Migration**
+
+### **Prerequisite**
+
+The following need to be true before you follow this guide:
+* AWS console logon
+* Access the DMS service from console
+* A region where STS has been activated
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: `Access the DMS service from console` should this be Access TO the DMS service from THE console

File: how-to-docs/dms-migration.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+### **DMS Setup** 
+
+Login to the AWS console using your auth, switch to a role with the correct access policies and verify you're in the right region. Next, select DMS from the services on the main dashboard to access to data migration home screen. Under the "Get started" section click on the "create migration" button then next to the Replication instance. You should see the following screen:
+
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: `to access to` to access the

File: how-to-docs/dms-migration.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+![Alt text](pics/dms-doc-1.png?raw=true)
+
+The following are the options and example answer for the replication instance:
+
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: an example

File: how-to-docs/examples/redis-temp.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
+{% set service_name='' %}
+{% set service_env='' %}
+{% set dns_name=service_name+'-'+service_env %}
+---
+name: {{ service_name }}-{{ service_env }}-rds
+---
```
Comment (Joseph-Irving) [inline how-to-docs/examples/redis-temp.yaml]: what it this file?

Comment (AwkwardBen) [inline how-to-docs/examples/redis-temp.yaml]: damn wrong template

**Pull request merged**